src/Testament/index.html

```diff
@@ -101,20 +101,29 @@
     grid-template-columns: auto auto;
     grid-template-rows: auto auto;
     }
-    .container .main-content .title {font-size: 10vw}
+    .container .main-content .title {
+        font-size: 10vw    
+    }
+    .container .content {
```

Comment:
We can try changing font formatting later. For the time being, just concentrate on padding thing.